DNN Platform/Library/Properties/AssemblyInfo.cs

```diff
@@ -56,4 +56,14 @@
 [assembly: InternalsVisibleTo("DotNetNuke.Tests.Web.Mvc")]
 [assembly: InternalsVisibleTo("DotNetNuke.Tests.Urls")]
 [assembly: InternalsVisibleTo("DotNetNuke.Tests.Professional")]
-[assembly: InternalsVisibleTo("DotNetNuke.SiteExportImport")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("DotNetNuke.SiteExportImport")]
+[assembly: InternalsVisibleTo("DotNetNuke.Web.DDRMenu")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
+[assembly: InternalsVisibleTo("Dnn.PersonaBar.Extensions")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
+[assembly: InternalsVisibleTo("DotNetNuke.Modules.Html")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
+[assembly: InternalsVisibleTo("DotNetNuke.Website.Deprecated")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
+[assembly: InternalsVisibleTo("Dnn.PersonaBar.UI")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
+[assembly: InternalsVisibleTo("Dnn.PersonaBar.Library")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
+[assembly: InternalsVisibleTo("DotNetNuke.Modules.Groups")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
+[assembly: InternalsVisibleTo("DotNetNuke.Modules.Journal")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
+[assembly: InternalsVisibleTo("DotNetNuke.Modules.RazorHost")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
+[assembly: InternalsVisibleTo("DotNetNuke.Website")] // Once Globals is refeactored to Dependency Injection we should be able to remove this
```

Comment:
The `Globals.DependencyProvider` (our DependencyInjection Container) is internal by design. However the entire platform doesn't fully support Dependency Injection and there are many areas where we need to manually resolve dependencies using the Service Locator Pattern

```
IMyCutomType myInstance = Globals.DependencyProvider.GetService<IMyCustomType>();
```

I am not sure the best way to approach this problem as to make this change I need all of those `InternalsVisibleTo` declarations otherwise we won't have access to the `DependencyProvider`.

@mitchelsellers and I have discussed in the past that we do not want to make the `DependencyProvider` public and it is internal to the platform.

Reply:
Gotcha, that works for me.